.infrastructure/00_main.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_version = "1.5.7"
+
+  backend "s3" {}
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.67.0"
```

Comment:
Here we could evaluate to use a [recent version](https://github.com/hashicorp/terraform-provider-aws/releases).

If you are going to change it, then be careful to also update the REDME 😄

Reply:
I set the version v5.16.2 and i've updated the README file

Reply:
to upgrade to version 5.16.2 you need to init terraform again
![image](https://github.com/pagopa/b2b-portals/assets/98483756/a435e676-027a-4bf1-bc31-56462de49aa8)